arrows/ffmpeg/tests/test_video_input_ffmpeg.cxx

```diff
@@ -478,12 +482,19 @@ TEST_F(ffmpeg_video_input, no_sync_metadata)
   // Open the video
   vif.open(video_path);
 
+  auto const& caps = vif.get_implementation_capabilities();
+  EXPECT_TRUE( caps.capability( kwiver::vital::algo::video_input::HAS_METADATA ) );
+
   kwiver::vital::timestamp ts;
 
   unsigned int frame_num = 0;
   while (vif.next_frame(ts))
   {
     auto md_vect = vif.frame_metadata();
+
+    EXPECT_TRUE(md_vect.size() > 0)
```

Comment:
Just out of curiosity, this is true solely because of adding the video name to each frame?

Reply:
Yes - for this video the only metadata is the video filename. There is no actual metadata stored in the video file.